docs/development.md

```diff
@@ -102,53 +109,53 @@ By default, running `go generate ./...` will compile a few different variations
 of the LLM library based on common CPU families and vector math capabilities,
```

Comment:
Change this to `make`? Technically, I guess this section applies to Windows as well.